OpenOversight/app/templates/base.html

```diff
@@ -122,7 +122,9 @@ <h5>Contact</h5>
           <p>
             <a href="http://twitter.com/TechBlocSEA" rel="noopener noreferrer" target="_blank"><i class="fa fa-twitter-square fa-3x social"></i></a>
             <a href="https://github.com/SeattleDSA/OpenOversight" rel="noopener noreferrer" target="_blank"><i class="fa fa-github fa-3x social"></i></a>
-            <a href="mailto:techblocsea@protonmail.com"><i class="fa fa-envelope-square fa-3x social"></i></a><br>
+            <a href="mailto:techblocsea@protonmail.com"><i class="fa fa-envelope-square fa-3x social"></i></a>
+            <a href="https://tech-bloc-sea.dev/" rel="noopener noreferrer" target="_blank"><i class="fa fa-globe fa-3x social"></i></a>
+            <br>
```

Comment:
Sorry I missed this earlier but for accessibility we'll want to add `aria-label`s to each of these.

Reply:
Very noob question but what should the values for those labels be?

Reply:
We could use `title` instead of `aria-label` I think. I'm still not sure, based on what I'm reading online the best would be to add visually-hidden text inside the anchor element. I think these would be appropriate:

| Link | Accessible text |
|--|--|
| Twitter | TechBloc Twitter |
| GitHub | OpenOversight GitHub |
| mailto | Doesn't need one I don't think cause the SR will just read the email address I believe, I'll confirm that |
| Directory | TechBloc Projects |

More information here: https://www.a11y-101.com/development/icons-and-links

We can go with `aria-label` for now and switch to visually-hidden (sr-only) text later.

The icons should all have `aria-hidden="true"` no matter what though.

Reply:
This is incredibly helpful, thank you!